projekt_do_pracy_na_labach/fiszki_backend/backend.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: example-deployment
+  namespace: lab
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: example
+  template:
+    metadata:
+      labels:
+        app: example
+    spec:
+      containers:
+      - name: backend-service
+        image: fiszki-backend:latest
+        imagePullPolicy: IfNotPresent
+        ports:
+        - containerPort: 8000
+
```

Comment:
Jak dodasz odwołanie się do persistent volume które stworzyłeś w bakcendzie to dam 3.5 :p